src/main/java/edu/cornell/kfs/module/ar/service/impl/CuCustomerAddressHelperServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package edu.cornell.kfs.module.ar.service.impl;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.integration.cg.ContractsAndGrantsBillingAward;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.module.ar.ArConstants.ContractsAndGrantsInvoiceDocumentCreationProcessType;
+
+import edu.cornell.kfs.module.ar.service.CuCustomerAddressHelperService;
+
+public class CuCustomerAddressHelperServiceImpl implements CuCustomerAddressHelperService {
+    
+    private static final Logger LOG = LogManager.getLogger();
+    
+    /*
+     * CU Customization: KFSPTS-26393
+     * When creationProcessType is BATCH, validate that the customer number on the award as part if the customer
+     * address composite key matches the customer number associated to the agency associated to the award.
+     */
+    public boolean agencyCustomerMatchesAwardCustomer(ContractsAndGrantsBillingAward award,
+            ContractsAndGrantsInvoiceDocumentCreationProcessType creationProcessType) {
```

Comment:
Should the check in the creation process type be done outside of this method?

Reply:
Based on the fact that this is modeled on base code I am ok with the code as is.